dist/test_data.py

```diff
@@ -64,8 +64,12 @@ def __ge__(self, other):
 workload_config = [
     Config('enable_tracking', 'true', r'''
         Enables tracking to perform validation''', type='boolean'),
+    Config('enable_timestamp', 'true', r'''
```

Comment:
I think workload_config should be config used by the workload generator, can we get a timestamp_config that has:
`enable_timetamps` and `timestamp_window_seconds`.